Sources/SwiftGenKit/Parsers/Strings/StringsEntry.swift

```diff
@@ -10,16 +10,15 @@ import PathKit
 extension Strings {
   struct Entry {
     let key: String
-    var keyStructure: [String] {
-        return key.components(separatedBy: CharacterSet(charactersIn: "."))
-    }
     let translation: String
     let types: [PlaceholderType]
+    let keyStructure: [String]
 
     init(key: String, translation: String, types: [PlaceholderType]) {
       self.key = key
       self.translation = translation
       self.types = types
+      keyStructure = Entry.explode(key: key)
```

Comment:
Note that I'm storing it as a constant, so that we don't have to explode/split the key each time (which happens once for each level).

Reply:
Oh indeed, missed that optimisation.

Probably because you didn't use `self.` like we do for other properties so it didn't make it obvious to me that this was referring to a property not a local variable 😉😉

Reply:
I avoid the use of self where it's not needed, so I get xcode warnings when I actually need it (strong/weak)